src/LOOPSUID

```diff
@@ -152,36 +150,38 @@ DONTEVAL@COMPILE DOCOPY
 (DEFINEQ
 
 (InitializeUIDs
-  [LAMBDA NIL                                                (* ; "Edited 21-Apr-88 11:38 by jrb:")
+  [LAMBDA NIL                                             (* ; "Edited 11-Mar-2022 18:41 by briggs")
+                                                             (* ; "Edited 21-Apr-88 11:38 by jrb:")
 
 (* ;;; "Initializes the UniqueIdentifer Generation System in DB.  Sets the global variables *UID-SESSION* and and *UID-COUNT*")
                                                              (* ; 
                                                              "Make sure that we have a valid time!")
-
     (DECLARE (GLOBALVARS \RCLKMILLISECOND \MY.NSHOSTNUMBER))
     (while (IGREATERP (IDATE MAKESYSDATE)
                   (IDATE)) do (ERROR "Time is not set! Call" 
                                      "(SETTIME %"dd-mmm-yy hh:mm:ss%") and type RETURN"))
                                                              (* ; 
                                                       "Compute *UID-SESSION* and DB.UIREC for today.")
-
-    (LET (date year month day monthCode startIndex nsHostNumber)
+    (LET (date separator year month day monthCode startIndex nsHostNumber)
          (SETQ nsHostNumber (OR (LISTP \MY.NSHOSTNUMBER)
                                 (LIST 'NSHOSTNUMBER (DAYTIME)
                                       0 0)))
-          
-          (* ;; "Set up hostNumbe from \MY.NSHOSTNUMBER which should always be set in Interlisp-D -- else compute a random one which should not be used by any one")
+
+         (* ;; "Set up hostNumbe from \MY.NSHOSTNUMBER which should always be set in Interlisp-D -- else compute a random one which should not be used by any one")
```

Comment:
This assertion isn't true -- that \MY.NSHOSTNUMBER is always set. I think it wasn't as well-established then that it was legitimate computer science to accept that randomly generated GUIDs or even those based on hash algorithms were actually "globally unique".
Why does LOOPS need GUIDs?